server/src/domain/repositories/communication.repository.ts

```diff
@@ -21,6 +21,10 @@ export enum ServerEvent {
   CONFIG_UPDATE = 'config:update',
 }
 
+export enum InternalEvent {
+  VALIDATE_CONFIG = 'validate_config',
```

Comment:
Can we make an event map too?

Reply:
Not really I think. The normal `emit` function only returns a boolean. The `emitAsync` function returns an array of `any`, so that could probably be typed, yes. But I don't know how we could separate them.